src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,56 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final int OPERATION_TYPE_COLUMN = 0;
+    private static final int AMOUNT_COLUMN = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int supplySum = 0;
+        int buySum = 0;
+        File fromFile = new File(fromFileName);
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFile));
+            String line = bufferedReader.readLine();
+            while (line != null) {
+                String[] splitLine = line.split(",");
+                if (splitLine[OPERATION_TYPE_COLUMN].equals(SUPPLY)) {
+                    supplySum += Integer.parseInt(splitLine[AMOUNT_COLUMN]);
+                } else if (splitLine[OPERATION_TYPE_COLUMN].equals(BUY)) {
+                    buySum += Integer.parseInt(splitLine[AMOUNT_COLUMN]);
+                }
+                line = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't open the file", e);
+        }
```

Comment:
you should split this block of code into two methods, one for reading from a file and the second one for calculation of needed parameters